### PR TITLE
Added Celebrity Problem

### LINKED ISSUE #1401: Celebrity Problem in C, C++, Java, Python & Go

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #1401 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Celebrity Problem in 5 languages.

#### Languages Used:
- C
- C++
- Java
- Python
- Go

#### Files Added:
- Celebrity_Problem.c
- Celebrity_Problem.cpp
- Celebrity_Problem.java
- Celebrity_Problem.py
- Celebrity_Problem.go

### CODE REVIEW

File: Celebrity_Problem/Celebrity_Problem.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+he knows noone.
+We need to find such a person, if present.
+
+There will be a matrix input, such that M[i][j] = 1, if ith person
+knows jth person, else 0.
+*/
+
+#include<stdio.h>
+
+int Celebrity (int Person[][100], int n) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.c]: Remove space after ``Celebrity``.

File: Celebrity_Problem/Celebrity_Problem.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    int start = 0;
+    int end = n - 1;
+    while (1) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.c]: Remove space after ``while``.

File: Celebrity_Problem/Celebrity_Problem.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+We need to find such a person, if present.
+
+There will be a matrix input, such that M[i][j] = 1, if ith person
+knows jth person, else 0.
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int Celebrity (int Person[][100], int n) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.cpp]: Fix

File: Celebrity_Problem/Celebrity_Problem.cpp [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    int start = 0;
+    int end = n - 1;
+    while (true) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.cpp]: Fix

File: Celebrity_Problem/Celebrity_Problem.go [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+knows jth person, else 0.
+*/
+
+package main
+
+import (
+    "fmt"
+)
+
+func Celebrity (Person [][]int, n int) int {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.go]: Fix

File: Celebrity_Problem/Celebrity_Problem.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+There will be a matrix input, such that M[i][j] = 1, if ith person
+knows jth person, else 0.
+*/
+
+import java.util.Scanner;
+
+class Solution {
+
+    public static int Celebrity (int Person[][], int n) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.java]: Fix

File: Celebrity_Problem/Celebrity_Problem.java [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        int start = 0;
+        int end = n - 1;
+        while (true) {
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.java]: Fix

File: Celebrity_Problem/Celebrity_Problem.py [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+In a group of N people, there is 1 person, who is known to everyone, but
+he knows noone.
+We need to find such a person, if present.
+
+There will be a matrix input, such that M[i][j] = 1, if ith person
+knows jth person, else 0.
+'''
+
+def Celebrity (Person, n):
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.py]: Here also

File: Celebrity_Problem/Celebrity_Problem.py [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    start = 0
+    end = n - 1
+    while (True):
```
Comment (marwahmanbir) [inline Celebrity_Problem/Celebrity_Problem.py]: Fix

**Comment by MastersAbh** [comment]
Done. Please review @marwahmanbir

**Pull request merged**